pkgs/applications/editors/vim/plugins/non-generated/bitbake-vim/default.nix

```diff
@@ -0,0 +1,21 @@
+{
+  fetchFromGitHub,
+  nix-update-script,
+  vimUtils,
+}:
+# The GitHub repository returns 404, which breaks the update script
+vimUtils.buildVimPlugin {
+  pname = "bitbake.vim";
+  version = "2021-02-06";
+
+  src = fetchFromGitHub {
+    owner = "sblumentritt";
+    repo = "bitbake.vim";
+    rev = "faddca1e8768b10c80ee85221fb51a560df5ba45";
+    sha256 = "1hfly2vxhhvjdiwgfz58hr3523kf9z71i78vk168n3kdqp5vkwrp";
+  };
+
+  meta.homepage = "https://github.com/sblumentritt/bitbake.vim/";
```

Comment:
doesnt seem reachable. It was nice to provide this thanks to nix cache but if we can't build from source and you are doing a cleanup anyway drop it. If you feel nice you can suggest an alternative (i dunno any).

Reply:
Should we redirect this to https://github.com/kergoth/vim-bitbake ?

Reply:
I think bitbake users should use https://github.com/openembedded/bitbake/tree/master/contrib/vim instead. If you feel motivated, you could change the src by that url instead